Bugsnag/Assets/Bugsnag/Editor/SymbolUpload/BugsnagAndroidSymbolUploader.cs

```diff
@@ -0,0 +1,246 @@
+            return true;
+        }
+
+        private bool MakeExecutable(string filePath)
+        {
+            if (Environment.OSVersion.Platform == PlatformID.Unix || Environment.OSVersion.Platform == PlatformID.MacOSX)
+            {
+                UnityEngine.Debug.Log($"Making the Bugsnag CLI executable with chmod on Unix/MacOS platform...");
+
+                var chmodProcess = new Process()
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = "chmod",
+                        Arguments = $"+x {filePath}",
+                        RedirectStandardOutput = true,
+                        RedirectStandardError = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true
+                    }
+                };
+
+                chmodProcess.Start();
+                string chmodOutput = chmodProcess.StandardOutput.ReadToEnd();
+                string chmodError = chmodProcess.StandardError.ReadToEnd();
+                chmodProcess.WaitForExit();
+
+                if (chmodProcess.ExitCode != 0)
+                {
+                    UnityEngine.Debug.LogError($"Failed to make the CLI executable. Error: {chmodError}");
+                    return false;
+                }
+                UnityEngine.Debug.Log($"Bugsnag CLI made executable successfully. Output: {chmodOutput}");
+            }
+
+            return true;
+        }
+
+        private bool RunBugsnagCliCommand(string cliPath, string arguments)
+        {
+            if (!File.Exists(cliPath))
+            {
+                UnityEngine.Debug.LogError($"Bugsnag CLI not found at {cliPath}");
+                return false;
+            }
+
+            UnityEngine.Debug.Log($"Running Bugsnag CLI tool with arguments: {arguments}");
+
+            var runCliProcess = new Process()
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = cliPath,
+                    Arguments = arguments,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    UseShellExecute = false,
+                    CreateNoWindow = true
+                }
+            };
+
+            runCliProcess.Start();
+            string runCliOutput = runCliProcess.StandardOutput.ReadToEnd();
+            string runCliError = runCliProcess.StandardError.ReadToEnd();
+            runCliProcess.WaitForExit();
+
+            UnityEngine.Debug.Log($"Bugsnag CLI process exited with code {runCliProcess.ExitCode}");
+
+            if (runCliProcess.ExitCode != 0)
+            {
+                UnityEngine.Debug.LogError($"Failed to run the Bugsnag CLI tool. Error: {runCliError} Output: {runCliOutput}");
+                return false;
+            }
+
+            UnityEngine.Debug.Log($"Bugsnag CLI tool executed successfully. Output: {runCliOutput}");
```

Comment:
Suggest having this message in the calling function and more generally - "Symbol files uploaded successfully")